tests/unit/spec/AdalSpec.js

```diff
@@ -586,6 +586,26 @@ describe('Adal', function () {
         expect(storageFake.getItem(adal.CONSTANTS.STORAGE.USERNAME)).toBeUndefined();
     });
 
+    it ('test decode with no padding', function () {
+        expect(adal._decode('YW55IGNhcm5hbCBwbGVhc3Vy')).toBe('any carnal pleasur');
```

Comment:
Please change the encoded string.  This one is on the edge of being inappropriate.  If this came from a test published elsewhere we should update that test as well.